tools/proofers/preview.js

```diff
@@ -689,6 +704,49 @@ $(function () {
             }
         }
 
+        // check that math delimiters \[ \], \( \) are correctly matched
+        function checkMath() {
+            var mathRe = /\\\[|\\\]|\\\(|\\\)/g;
+            var result;
+            var tag, openTag = null;
```

Comment:
why set to null, I only see falsy checks, nothing explicitly to null.

Reply:
I suppose you are right, I will change it.